src/bundle/Resources/views/parts/menu/top_menu_2nd_level.html.twig

```diff
@@ -34,6 +34,6 @@
     </a>
 {% endblock %}
 
-{% block spanElement %}
-    {{ block('linkElement') }}
+{% block span_element %}
```

Comment:
Isn't that block from KNP? Can we change this? /cc @webhdx

Reply:
I don't think we can change it. This overrides the block from KNP.

Reply:
Ok, I fixed this one @dew326 @webhdx